### ISSUE TITLE
Content switcher: Style tab

### ISSUE THREAD

**Issue by laurenmrice** [carbon-design-system/carbon-website#3618]
Ref original issue: https://github.com/carbon-design-system/carbon-website/issues/3364
Ref Style tab PDF comments: https://ibm.ent.box.com/file/1000661068886?s=7ncuk4he7chgcehb0a3fm6jccqqikd1k

----

## Style tab

**Icon content switcher**: Add content and images to sections that apply.
**Interactive states:**Missing Unselected tab with focus state.

**Comment by Huixh111** [comment]
o
